srcpkgs/lldpd/files/lldpd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec lldpd ${OPTS:- -d}
```

Comment:
this logs to stderr, no? which would leak up to runsvdir? Probably want to redirect to stdout and set up a logger.

Reply:
Good catch. I've redirected stderr to stdout and linked in vlogger. There seemed to be three distinct ways to do logging, so I'm not sure if this is the preferred method or not.